src/rollback_to_stable/rts_history.c

```diff
@@ -60,10 +63,10 @@ __wt_rts_history_delete_hs(WT_SESSION_IMPL *session, WT_ITEM *key, wt_timestamp_
             __wt_verbose_level_multi(session, WT_VERB_RECOVERY_RTS(session), WT_VERBOSE_DEBUG_3,
               WT_RTS_VERB_TAG_HS_UPDATE_REMOVE
               "deleting history store update with stop_timestamp=%s greater than "
-              "stable_timestamp=%s, time_window=%s",
+              "stable_timestamp=%s, time_window=%s, btree_id=%" PRIu32,
```

Comment:
How about changing the sentence to include `btree_id` before the `stop_timestamp`?

Reply:
Thanks, done in 70f9246